domains/api/grpc/client.go

```diff
@@ -65,3 +73,30 @@ func encodeDeleteUserRequest(_ context.Context, grpcReq interface{}) (interface{
 		Id: req.ID,
 	}, nil
 }
+
+func (client domainsGrpcClient) RetrieveDomainStatus(ctx context.Context, in *grpcDomainsV1.RetrieveDomainStatusReq, opts ...grpc.CallOption) (*grpcDomainsV1.RetrieveDomainStatusRes, error) {
```

Comment:
We can have some thing like  `RetrieveEntity` which can returns domain id, name and status instead of `RetrieveDomainStatus` . 
What do you think ?

Reply:
I think this okay.